From 8754ebe5cf56ad8314bd80a4ed5461f500d2fb7e Mon Sep 17 00:00:00 2001
From: Viresh Kumar <viresh.kumar@linaro.org>
Date: Wed, 7 Oct 2020 11:07:44 +0530
Subject: [PATCH] syscalls: semop: Increase timeout for semop03

Sometimes the timeout happens before the signal is received and it
causes the test to fail with following error:

 semop03.c:55: TFAIL: unexpected failure: EAGAIN/EWOULDBLOCK (11)

Fix this by increasing the timeout to a sufficiently large value.

Reported-by: Rachel Sibley <rasibley@redhat.com>
Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
Acked-by: Jan Stancek <jstancek@redhat.com>
Reviewed-by: Li Wang <liwang@redhat.com>
---
 testcases/kernel/syscalls/ipc/semop/semop03.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/kernel/syscalls/ipc/semop/semop03.c b/testcases/kernel/syscalls/ipc/semop/semop03.c
index 89603f19d..3a79ed38a 100644
--- a/testcases/kernel/syscalls/ipc/semop/semop03.c
+++ b/testcases/kernel/syscalls/ipc/semop/semop03.c
@@ -71,7 +71,7 @@ static void setup(void)
 	semop_supported_by_kernel(tv);
 
 	timeout.type = tv->ts_type;
-	tst_ts_set_sec(&timeout, 0);
+	tst_ts_set_sec(&timeout, 2);
 	tst_ts_set_nsec(&timeout, 10000000);
 
 	SAFE_SIGNAL(SIGHUP, sighandler);
-- 
2.21.3

